fzf-git.sh

```diff
@@ -180,7 +180,7 @@ _fzf_git_hashes() {
     --bind 'ctrl-d:execute:grep -o "[a-f0-9]\{7,\}" <<< {} | head -n 1 | xargs git diff > /dev/tty' \
     --color hl:underline,hl+:underline \
     --preview 'grep -o "[a-f0-9]\{7,\}" <<< {} | head -n 1 | xargs git show --color=always' "$@" |
-  grep -o "[a-f0-9]\{7,\}"
+  grep -o "[a-f0-9]\{7,\}" | head -n 1
```

Comment:
We can't do this because we should be able to select multiple hashes (`<tab>`). We need a way to extract only one hash from each line.

Reply:
Nice! I didn't consider that situation!
Thank you!